zerver/webhooks/freshstatus/view.py

```diff
@@ -0,0 +1,164 @@
+from typing import Any, Dict, List
+
+import dateutil.parser
+from django.http import HttpRequest, HttpResponse
+from django.utils.translation import ugettext as _
+
+from zerver.decorator import REQ, has_request_variables, webhook_view
+from zerver.lib.actions import send_rate_limited_pm_notification_to_bot_owner
+from zerver.lib.response import json_error, json_success
+from zerver.lib.send_email import FromAddress
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+MISCONFIGURED_PAYLOAD_ERROR_MESSAGE = """
+Hi there! Your bot {bot_name} just received a Freshstatus payload that is missing
+some data that Zulip requires. This usually indicates a configuration issue
+in your Freshstatus webhook settings. Please make sure that you provide all the required parameters
+when configuring the Freshstatus webhook. Contact {support_email} if you
+need further help!
+"""
```

Comment:
Ideally, we'd be not need custom text of this form for individual webhooks; might be worth thinking about how to make a function return this string.

Reply:
@timabbott This is how it is for all the other integrations, for the the reason that, this is customized slightly for certain integrations to give more of a pointer of what the issue could be. For example the one in the Zabbix integration that was added back in #11081 by @eeshangarg points the user towards ensuring that script you have install on zabbix is setup correctly. And it is only used on a handful of integrations where you can customize the payload and we need to ensure it is set up correctly following our instructions.

Reply:
We could add a function to create this, which we could call, but it wouldn't allow us to create more informing messages to be sent to the users.

Reply:
Yeah, it's not a blocker, just a thing to think about if there's an opportunity to avoid duplication.